src/Bundle/DependencyInjection/Driver/AbstractDriver.php

```diff
@@ -120,7 +121,7 @@ protected function addFactory(ContainerBuilder $container, MetadataInterface $me
 
         $typehintClasses = array_merge(
             $factoryInterfaces,
-            [$factoryClass],
+            [$factoryClass, LegacyFactoryInterface::class],
```

Comment:
Is the same necessary for `LegacyTranslatableFactoryInterface`?

Reply:
done, thx